backend/src/api/handlers/webSocket.go

```diff
@@ -26,11 +36,34 @@ func ConnectWS(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	//Find random words from slices
+	rand.Seed(time.Now().UnixNano())
+	ranAdj := rand.Intn(1333)
+	//ranNoun := rand.Intn(1524)
+	adj := adjectives[ranAdj]
+	//noun := nouns[ranNoun]
+
 	player := structs.Player{
-		Ws:   wsConnection,
-		Name: fmt.Sprintf("bobby%d", rand.Intn(100)),
+		Ws: wsConnection,
+		//Name: fmt.Sprintf(("%s%s"), strings.TrimSpace(strings.Title(adj)), strings.Title(noun)),
+		Name: fmt.Sprintf(("%sBobby"), strings.TrimSpace(strings.Title(adj))),
 	}
 
 	mines.Players = append(mines.Players, &player)
 	mines.SocketListener(&player)
 }
+
+//Read file
+func readFromFile(filename string, lines int) []string {
+	file, err := os.Open(filename)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer file.Close()
+
+	readWord, _ := ioutil.ReadFile(filename)
+	allWord := string(readWord)
+	listWord := strings.Split(allWord, "\n")
```

Comment:
Handle error and just log fatal plis. I might be a hypocrite when saying this...